crates/ethcore/src/miner/pool_client.rs

```diff
@@ -155,9 +155,8 @@ where
     ) -> Result<SignedTransaction, transaction::Error> {
         self.engine
             .verify_transaction_basic(&tx, &self.best_block_header)?;
-        let tx = self
-            .engine
-            .verify_transaction_unordered(tx, &self.best_block_header)?;
```

Comment:
I would say yes. 
This part of code is substituted with "let tx = SignedTransaction::new(tx)?;", which is what "verify_transaction_unordered" function was actually doing before this PR.
By doing this substitution I was aiming to avoid base_fee checking for transactions currently being imported into tx pool (since after this PR verify_transaction_unordered does a base_fee checking). 
Maybe there is a more elegant way to achieve the same.

Reply:
You are right, i was suspicious but didn't check it.